LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Rakuten, Inc.
+(C) Rakuten Group, Inc.
```

Comment:
Why capitalized C?
Was that in a spec?

Reply:
Yes, but I'll confirm.

Reply:
It was confirmed to use (C) instead of (c).